src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.File.Create.cs

```diff
@@ -10,7 +10,7 @@ namespace System.IO.Tests
 {
     public class File_Create_Tests : FileSystemWatcherTest
     {
-        [Fact]
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsSubsystemForLinux))] // https://github.com/Microsoft/BashOnWindows/issues/216
```

Comment:
I'm getting notifications for all of the BashOnWindows issues, and also keeping a list of them in #11057.  We could track these with individual CoreFx issues, but I'm not sure it's worth the overhead.  Do you think this would have significant value?